doc/content/reference/id-eui-constraints/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ID and EUI Constraints"
+description: ""
+---
+
+This reference explains how {{% tts %}} deals with IDs and EUIs, including regular expression requirements, uniqueness, and how deletion is handled.
+
+<!--more-->
+
+## Requirements of an ID or EUI
+
+An ID or EUI in {{% tts %}} must:
+
+- Have a length of 3-36 characters
```

Comment:
I'd say between 3 and 36 characters.

Reply:
FYI: There is also an open issue about lowering the minimum from 3 to 2: https://github.com/TheThingsNetwork/lorawan-stack/issues/3723